s/toolbox/hamster-html/hamster.test.ts

```diff
@@ -0,0 +1,28 @@
+
+import { html, render } from "./html.js";
+import { Suite, assert, expect } from "cynic";
+
+export default <Suite>{
+	"santization": {
+		"sanitization occurs by default": async () => {
+			const input = html`${"<div></div>"}`
+			const output = "&lt;div&gt;&lt;/div&gt;"
+
+			assert(render(input)===output,"sanitization does not occur")
+		},
+		"nested html functions must not be sanitized": async () => {
```

Comment:
maybe this test belongs under the `nesting` group?

Reply:
okay